lib/auth/clt.go

```diff
@@ -419,6 +420,29 @@ func (c *Client) GetClusterCACert() (*LocalCAResponse, error) {
 	return &localCA, nil
 }
 
+// GetClusterCACerts returns the CA certs for the cluster without signing keys.
+func (c *Client) GetClusterCACerts() (*ClusterCACertsResponse, error) {
+	var caCerts ClusterCACertsResponse
+	out, err := c.Get(c.Endpoint("cacerts"), url.Values{})
+	if err != nil {
```

Comment:
Can we check specifically for "not implemented" error here? That's what we do in other places IIRC if you add the new methods to GRPC server according to my other comment.

Reply:
Yes, that's easier with the gRPC method, it does that in the latest commit.